### PR TITLE
fix global port in script

### PR DESCRIPTION
**Pull request by Jackmin801**
The global PGs currently use ascending port numbers so it needs to have some distance from the fsdp port number

### CODE REVIEW

File: scripts/simulate_multi_node_diloco.sh [hunk 1]
```diff
@@ -53,17 +53,17 @@ trap cleanup SIGINT
 mkdir -p logs
 
 export GLOBAL_ADDR=localhost
-export GLOBAL_PORT=10000
+export GLOBAL_PORT=1234
```
Comment (samsja) [inline scripts/simulate_multi_node_diloco.sh]: can we make this port en env var that default to 1234 ? We probably don't know in advance which port are opem

File: scripts/simulate_multi_node_diloco.sh [hunk 2]
```diff
@@ -53,17 +53,17 @@ trap cleanup SIGINT
...
 export GLOBAL_WORLD_SIZE=$N
 
 for i in $(seq 0 $(($N - 1 )))
 do
     > logs/log$i
-    GLOBAL_UNIQUE_ID=$i GLOBAL_RANK=$i CUDA_VISIBLE_DEVICES=$(get_cuda_devices $NUM_GPU $i) uv run torchrun --nproc_per_node=$NUM_GPU --node-rank 0 --rdzv-endpoint localhost:$((10001 + $i)) --nnodes=1  $@  > logs/log$i 2>&1 &
+    GLOBAL_UNIQUE_ID=$i GLOBAL_RANK=$i CUDA_VISIBLE_DEVICES=$(get_cuda_devices $NUM_GPU $i) uv run torchrun --nproc_per_node=$NUM_GPU --node-rank 0 --rdzv-endpoint localhost:$((10001 + $i)) --nnodes=1  $@  > logs/log$i.log 2>&1 &
```
Comment (samsja) [inline scripts/simulate_multi_node_diloco.sh]: lets make 10001 an en var as well

File: scripts/simulate_multi_node_diloco.sh (hunk 1 above)
Comment (Jackmin801) [reply scripts/simulate_multi_node_diloco.sh]: yea makes sense :)

File: scripts/simulate_multi_node_diloco.sh (hunk 2 above)
Comment (Jackmin801) [reply scripts/simulate_multi_node_diloco.sh]: 🫡

**Review by samsja — APPROVED** [review]

**Pull request merged**